pkg/builtinchecks/yamls/privilege-escalation.yaml

```diff
@@ -0,0 +1,7 @@
+name: "privilege-escalation-container"
+description: "Alert on containers with allowing privilege escalation"
+remediation: "Ensure containers do not allow privilege escalation."
```

Comment:
Link to documentation, make rationale more detailed

Reply:
done.